sns/parity_math.c

```diff
@@ -382,12 +401,25 @@ static int vandmat_norm(struct m0_matrix *m)
 
 	return 0;
 }
-#endif /* __KERNEL__ */
+#endif /* RS_ENCODE_ENABLED */
+
+#if ISAL_ENCODE_ENABLED
+static inline void parity_math_validate(struct m0_parity_math *math)
```

Comment:
Usually, we can this xxx_invariant().
And _0C() can be used to verify conditions.
So this may look like this:
```
static bool parity_math_validate(struct m0_parity_math *math)
{
        return _0C(math != NULL) && _0C(math->pmi_data_count >= 1) &&
                   _0C(math->pmi_parity_count >= 1) &&
                   _0C(math->pmi_data_count >= math->pmi_parity_count) &&
                   _0C(math->pmi_data_count <= SNS_PARITY_MATH_DATA_BLOCKS_MAX);
}
```

Reply:
Done